<launch>
  <include file="$(find ar_pose)/launch/ar_pose_multiple_single.launch" />

  <arg name="debug" default="false"/>
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -rv -e gdb -ex run -args"/>
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>

  <node name="ARSynchronize" pkg="ar_pose" type="ar_synchronize" respawn="false" output="screen" launch-prefix="$(arg launch_prefix)">
    <rosparam command="load" file="$(find ar_pose)/config/synchronize.yaml"/>
  </node>

</launch>

